msgid ""
msgstr ""
"Project-Id-Version: univention-management-console-module-join\n"
"Report-Msgid-Bugs-To: packages@univention.de\n"
"POT-Creation-Date: 2020-09-25 01:15+0200\n"
"PO-Revision-Date: 2012-04-23 15:37+0200\n"
"Last-Translator: Univention GmbH <packages@univention.de>\n"
"Language-Team: Univention GmbH <packages@univention.de>\n"
"Language: de\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: umc/python/join/__init__.py:399 umc/python/join/__init__.py:450
msgid "A join process is already running."
msgstr "Ein Join-Vorgang läuft bereits."

#: umc/python/join/__init__.py:430 umc/python/join/__init__.py:483
#, python-format
msgid "An unexpected error occurred: %s"
msgstr "Ein unerwarteter Fehler ist aufgetreten: %s"

#: umc/python/join/__init__.py:464
msgid "Authentication"
msgstr "Authentifizierung"

#: umc/python/join/__init__.py:407 umc/python/join/__init__.py:454
msgid ""
"Currently, software is being installed or uninstalled. Join scripts should "
"not be run right now."
msgstr ""
"Momentan wird Software installiert oder deinstalliert. Join-Skripte sollten "
"jetzt nicht ausgeführt werden."

#: umc/python/join/__init__.py:411
msgid "Domain join"
msgstr "Domänenbeitritt"

#: umc/python/join/__init__.py:324
#, python-format
msgid "Error: %s"
msgstr "Fehler: %s"

#: umc/python/join/__init__.py:231
#, python-format
msgid "Executing join script %s"
msgstr "Ausführung des Join-Skripts %s"

#: umc/python/join/__init__.py:230
msgid "Executing join scripts"
msgstr "Ausführung der Join-Skripte"

#: umc/python/join/__init__.py:107
msgid "Initializing"
msgstr "Initialisiere"

#: umc/python/join/__init__.py:403
msgid "Invalid server role! A Primary Directory Node cannot be joined."
msgstr ""
"Ungültige Serverrolle. Ein Primary Directory Node kann nicht gejoined "
"werden."

#: umc/python/join/__init__.py:75
msgid ""
"No DNS record for the Primary Directory Node was found. This might be a "
"problem with the configured DNS server. Please make sure the DNS settings "
"are correct."
msgstr ""
"Es wurde kein DNS Eintrag für den Primary Directory Node gefunden. Es "
"könnte ein Problem mit den konfigurierten DNS-Servern vorliegen. Bitte "
"stellen Sie sicher, dass die DNS-Einstellungen korrekt sind."

#: umc/python/join/__init__.py:258
msgid ""
"Some join scripts could not be executed. More details can be found in the log "
"file <i>/var/log/univention/join.log</i>.<br/>Please retry to execute the join "
"scripts after resolving any conflicting issues."
msgstr ""
"Einige Join-Skripte konnten nicht ausgeführt werden. <br/>Weitere "
"Informationen können in der Logdatei <i>/var/log/univention/join.log</i> "
"gefunden werden.<br/>Bitte die Ausführung der Join-Skripte nach Lösen aller "
"Konflikte erneut durchführen."

#: umc/python/join/__init__.py:95
msgid "The entered FQDN is not a valid value"
msgstr "Die eingegebene FQDN ist kein gültiger Wert."

#
#: umc/python/join/__init__.py:78
msgid ""
"The lookup of the Primary Directory Node record timed out. There might be a "
"problem with the configured DNS server. Make sure the DNS server is up and "
"running or check the DNS settings."
msgstr ""
"Eine Zeitüberschreitung beim Suchen des Primary Directory Node DNS-"
"Eintrages ist aufgetreten. Es könnte ein Problem mit den konfigurierten DNS-"
"Servern vorliegen. Bitte stellen Sie sicher, dass die DNS-Server erreichbar "
"sind oder überprüfen Sie die DNS-Einstellungen."

#: umc/python/join/__init__.py:219
#, python-format
msgid ""
"The system join process could not be completed:<br/><br/><i>%s</i><br/><br/> "
"More details can be found in the log file <i>/var/log/univention/join.log</"
"i>.<br/>Please retry after resolving any conflicting issues."
msgstr ""
"Der Join-Vorgang konnte nicht durchgeführt werden:<br/><br/><i>%s</i><br/"
"><br/> Weitere Informationen können in der Logdatei <i>/var/log/univention/"
"join.log</i> gefunden werden.<br/>Bitte den Join-Vorgang nach Lösen aller "
"Konflikte erneut durchführen."

#: umc/python/join/__init__.py:425 umc/python/join/__init__.py:478
msgid "finished..."
msgstr "beendet..."

#: umc/python/join/__init__.py:310
msgid ""
"The join process could not be executed. More details can be found in the log file "
"<i>/var/log/univention/join.log</i>.<br/>Please retry to join the system after "
"resolving any conflicting issues."
msgstr ""
"Der Join-Vorgang konnte nicht durchgeführt werden. <br/>Weitere Informationen "
"können in der Logdatei <i>/var/log/univention/join.log</i> gefunden werden."
"<br/>Bitte die Ausführung der Join-Skripte nach Lösen aller Konflikte erneut "
"durchführen. "
